lib/data/model/invitation/generate_invitation_link_response.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'generate_invitation_link_response.g.dart';
+
+@JsonSerializable()
+class GenerateInvitationLinkResponse with EquatableMixin {
+  @JsonKey(name: "link")
+  final String? link;
```

Comment:
please request `id` also in `generate link` to make sure we can follow up the status

Reply:
should be non-null String

Reply:
still not update non-null string.